Project.toml

```diff
@@ -25,7 +25,7 @@ ZygoteRules = "700de1a5-db45-46bc-99cf-38207098b444"
 [compat]
 AbstractFFTs = "0.5"
 ArrayLayouts = "0.1, 0.2, 0.3"
-ChainRules = "0.7.0"
+ChainRules = "0.7"
```

Comment:
These should mean the same thing.

Reply:
Ah, I'd just misunderstood; the `.0` is redundant but it's not doing any harm. Thanks @KristofferC